Buildings/Fluid/HeatExchangers/DXCoils/BaseClasses/CoilDefrostCapacity.mo

```diff
@@ -0,0 +1,257 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.BaseClasses;
+block CoilDefrostCapacity
+  "Calculates defrost curve value at given temperature and mass flow rate"
+  extends Modelica.Blocks.Icons.Block;
+
+  replaceable package MediumA = Modelica.Media.Interfaces.PartialMedium
+    "Fluid medium package";
+
+  parameter Real tDefRun(
+    final unit="1",
+    displayUnit="1") = 0.5
+    "Time period for which defrost cycle is run"
```

Comment:
That should probably be time fraction, not time period, as a period has units of time?